tests/rptest/tests/debug_bundle_test.py

```diff
@@ -14,14 +14,23 @@
 from ducktape.utils.util import wait_until
 from rptest.services.admin import Admin, DebugBundleStartConfig, DebugBundleStartConfigParams
 from rptest.services.cluster import cluster
+from rptest.services.redpanda import LoggingConfig
 from rptest.tests.redpanda_test import RedpandaTest
 
+log_config = LoggingConfig('info', logger_levels={'admin_api_server': 'trace'})
```

Comment:
suggestion: maybe also increase the `debug-bundle-service` logger to trace as well

Reply:
Done